source/kubernetes/quickstart.rst

```diff
@@ -237,7 +260,7 @@ created using the dashboard method.
 
 .. Note::
 
-  Templates in the v1.20.x series onward need to specify the additional label:
+  Templates in the v1.30.x series onward need to specify the additional label:
```

Comment:
global search and replace here. Probably just drop this line.

Reply:
Yeah - now that 1.20 is long past, we should just remove the version and state that all templates need this if you want to create a publicly accessibly K8s API.